src/hexen/p_enemy.c

```diff
@@ -528,25 +528,29 @@ boolean P_LookForPlayers(mobj_t * actor, boolean allaround)
     player_t *player;
     angle_t an;
     fixed_t dist;
+    int i = 0; // For breaking vanilla infinite loop.
```

Comment:
A more descriptive name would be better. It's not really a loop index since it's reset during the loop.

Reply:
It never even claimed to be a loop index. In fact it's a loop counter. But admittedly, `i` is probably the most generic name of all.